cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
+    }
+})
+
+// delete an account 
+y.scriptName("connectme")
+y.command({
+    command: 'delete',
+    describe: 'This can only be done by the logged in user',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+    },
+    handler(argv) {
+        deleteuser(argv)
+    }
+})
+
+// get user by username
+y.scriptName("connectme")
+y.command({
+    command: 'get-user',
+    describe: 'get an user by username',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+    },
+    handler(argv) {
+        getuser(argv)
+    }
+})
+
+// update account
+y.scriptName("connectme")
+y.command({
+    command: 'update-user',
+    describe: 'update & existen account',
+    builder: {
+        userId: {
+            describe: 'and userID',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'password of the account',
+            demandOption: true,
+            type: 'string'
+        },
+        confirmPassword: {
+            describe: 'confirm the password',
+            demandOption: true,
+            type: 'string'
+        },
+        username: {
+            describe: 'and username',
+            demandOption: true,
+            type: 'string'
+        },
+        firstName: {
+            describe: 'your frist name',
+            demandOption: true,
+            type: 'string'
+        },
+        lastName: {
+            describe: 'your lastname',
+            demandOption: true,
+            type: 'string'
+        },
+        email: {
+            describe: 'your email',
+            demandOption: true,
+            type: 'string'
+        },
+        bio: {
+            describe: 'your biogaphy',
+            demandOption: true,
+            type: 'string'
+        },
+        website: {
+            describe: '...',
+            demandOption: true,
+            type: 'string'
+        },
+        location: {
+            describe: 'your actual location',
+            demandOption: true,
+            type: 'string'
+        },
+        picture: {
+            describe: '...',
+            demandOption: true,
+            type: 'string'
+        },Birthday:{
+            describe: 'User birthday',
+            type: 'string'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        updateuser(argv)
+    }
+})
+
+//get post command
+y.scriptName("connectme")
+y.command({
+    command: 'get-post',
+    describe: 'gets a post',
+    builder: {
+        post: {
+            describe: 'Post to get',
+            demandOption: true,
+            type: 'string'
+        },
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        getpost(argv)
+    }
+})
+//Post command
+y.scriptName("connectme")
+y.command({
+    command: 'post',

```

Comment:
if the command creates a post, maybe we should rename it to `create-post` instead of just `post`

Also when we create a post it belongs to a username, so we also need to pass the username that the post belongs to

Reply:
done

Reply:
I cannot see the change though